extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/runtime/LiquibaseMongodbConfig.java

```diff
@@ -15,6 +15,13 @@
 @ConfigRoot(name = "liquibase-mongodb", phase = ConfigPhase.RUN_TIME)
 public class LiquibaseMongodbConfig {
 
+    /**
+     * Flag to enable / disable Liquibase.
+     *
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
```

Comment:
Same question as the Flyway extension

Reply:
Acts as a flag to turn the job off, when we externalize it.